addons/website_project_issue/models/project_issue.py

```diff
@@ -1,13 +1,29 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, exceptions, models, _
+from odoo import api, fields, exceptions, models
 
 
 class Issue(models.Model):
     _name = "project.issue"
     _inherit = ['project.issue']
 
+    main_attachment_ids = fields.One2many('ir.attachment', compute='_compute_attachment_ids', string="Main Attachments",
```

Comment:
what do you prefer ? attachment_on_object_and_not_from_messages_ids ?

The purpose is to display the attachment from the browse record , and not only the attachment in messages that are dipslayed by the website_mail widget.